src/Controls/src/Core/AnimationExtensions.cs

```diff
@@ -138,9 +138,11 @@ public static bool AbortAnimation(this IAnimatable self, string handle)
 		}
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/AnimationExtensions.xml" path="//Member[@MemberName='Animate&lt;T&gt;'][1]/Docs" />
+#pragma warning disable CS1573 // Parameter has no matching param tag in the XML comment (but other parameters do)
```

Comment:
Technically we should fix the doc - maybe it is the wrong index as I see a `1`. Maybe we renamed a param and did not update the xml files?